src/libs/WorkspacesSettingsUtils.ts

```diff
@@ -195,14 +206,14 @@ function getWorkspacesBrickRoads(): Record<string, BrickRoad> {
 /**
  * @returns a map where the keys are policyIDs and the values are truthy booleans if policy has unread content
  */
-function getWorkspacesUnreadStatuses(): Record<string, boolean> {
-    if (!allReports) {
+function getWorkspacesUnreadStatuses(reports: OnyxCollection<Report>): Record<string, boolean> {
```

Comment:
All good ?

Reply:
Same here, this file has all reports already and the only caller is passing all reports, so there's no need to pass this param

Reply:
See [above (comment)](https://github.com/Expensify/App/pull/40197/files#r1567797163), it's for the same reason as detailed there.

**TLDR:** Using local action file's `allReports` would create a delay in the UI since `Onyx.connect` is decoupled from component's lifecycle.